modules/high-sev-security-policies.adoc

```diff
@@ -18,12 +18,13 @@ The following table lists the default security policies in {product-title} that
 [options="header"]
 |====
 |Life cycle stage |Name |Description | Status
-|Build or Deploy |Fixable CVSS >= 6 and Privileged |Alerts when deployments run in privileged mode with fixable vulnerabilities that have a CVSS of at least 6. |Enabled
-|Build or Deploy |Fixable CVSS >= 7 |Alerts when deployments with fixable vulnerabilities have a CVSS of at least 7. |Enabled
+|Build or Deploy |Fixable CVSS >= 7 |Alerts when deployments with fixable vulnerabilities have a CVSS of at least 7. |Disabled
 |Build or Deploy |Fixable Severity at least Important |Alerts when deployments with fixable vulnerabilities have a severity rating of at least Important. |Enabled
 |Build or Deploy |Secure Shell (ssh) Port Exposed in Image |Alerts when deployments expose port 22, which is commonly reserved for SSH access. |Enabled
 |Deploy |Emergency Deployment Annotation |Alerts when deployments use the emergency annotation, such as "admission.stackrox.io/break-glass":"ticket-1234" to circumvent StackRox Admission Controller checks. |Enabled
 |Deploy |Environment Variable Contains Secret |Alerts when deployments have environment variables that contain 'SECRET'. |Enabled
+|Deploy |Fixable CVSS >= 6 and Privileged |Alerts when deployments run in privileged mode with fixable vulnerabilities that have a CVSS of at least 6. |Disabled
```

Comment:
Technically this is only disabled if you do a fresh install (as of 3.72). If you upgrade from 71 or below, it _might_ be enabled (as it used to be default enabled and we don't change it for upgrades). Not sure if it's worth calling out though

Reply:
Good to know. I can update the text to something like, "Disabled in version 3.72.0 and later". WDYT? @theencee

Reply:
"Disabled by default in version 3.72.0 and later"? But yes that works!